### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by MichelleSwafford**
## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [ ] *if updating templates, it is helpful to post a screenshot of what the changes look like.*
* [ ] *if updating templates, ensure that your parameters and steps have meaningful names.*
* [ ] *if adding new templates, ensure that your parameters and columns have display names.*
* [ ] *if adding new templates, or changing items in a gallery, it's helpful to post a screenshot of what the gallery looks like now*

### CODE REVIEW

File: Workbooks/Azure Active Directory Hybrid Auth Staged Rollout/Hybrid Auth Staged Rollout/StagedRollout.workbooks.txt [hunk 1]
```diff
@@ -0,0 +1,287 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Staged Rollout users/groups and sign-ins\r\n"
```
Comment (KranthiKiranerusu) [inline Workbooks/Azure Active Directory Hybrid Auth Staged Rollout/Hybrid Auth Staged Rollout/StagedRollout.workbooks.txt:7]: Sentence casing, plesae make it readable.

File: Workbooks/Azure Active Directory Hybrid Auth Staged Rollout/Hybrid Auth Staged Rollout/StagedRollout.workbooks.txt [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+      "content": {
+        "json": "# Staged Rollout users/groups and sign-ins\r\n"
+      },
+      "name": "text - 10"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "AuditLogs \r\n| where OperationName contains 'feature'\r\n        and OperationName contains 'user'\r\n        and OperationName contains 'add'\r\n| where TimeGenerated > ago(30d) \r\n| make-series count(OperationName) \r\n    on TimeGenerated in range(ago(30d), now(), 1d) by OperationName\r\n\r\n\r\n\r\n",
```
Comment (KranthiKiranerusu) [inline Workbooks/Azure Active Directory Hybrid Auth Staged Rollout/Hybrid Auth Staged Rollout/StagedRollout.workbooks.txt:15]: has operator is more performant

Comment (KranthiKiranerusu) [inline Workbooks/Azure Active Directory Hybrid Auth Staged Rollout/Hybrid Auth Staged Rollout/StagedRollout.workbooks.txt:15]: Consider passing in time as a parameter, please take a look at the other workbooks for examples

File: Workbooks/Azure Active Directory Hybrid Auth Staged Rollout/Hybrid Auth Staged Rollout/StagedRollout.workbooks.txt [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+            "sortOrder": 2
+          }
+        ]
+      },
+      "customWidth": "50",
+      "name": "query - 6"
+    }
+  ],
+  "fallbackResourceIds": [
+    "/subscriptions/67f1e79b-7036-4a5a-9308-0de82af015eb/resourcegroups/jartek/providers/microsoft.operationalinsights/workspaces/jarteklogs"
```
Comment (KranthiKiranerusu) [inline Workbooks/Azure Active Directory Hybrid Auth Staged Rollout/Hybrid Auth Staged Rollout/StagedRollout.workbooks.txt:284]: remove this

**Review by KranthiKiranerusu — CHANGES_REQUESTED** [review]
Please address the comments.

**Comment by gardnerjr** [comment]
@MichelleSwafford when you create the PR, though, you should explain what you're doing.  the PR has title "add files via upload" and none of the description is filled in.  you should edit all of those to explain what you're doing, and the default content even has suggestions about what you should fill in.

**Pull request closed without merging**